src/features/HelpCompletion.ts

```diff
@@ -113,20 +124,20 @@ class TriggerFinder {
 }
 
 class HelpCompletionProvider {
-    private triggerFinderBlockComment: TriggerFinder;
     private triggerFinderLineComment: TriggerFinder;
     private lastChangeText: string;
     private lastChangeRange: Range;
     private lastDocument: TextDocument;
     private langClient: LanguageClient;
+    private settings: Settings.ISettings;
 
     constructor() {
-        this.triggerFinderBlockComment = new TriggerFinder("<#");
         this.triggerFinderLineComment = new TriggerFinder("##");
+        this.settings = Settings.load();
     }
 
     public get triggerFound(): boolean {
-        return this.triggerFinderBlockComment.found || this.triggerFinderLineComment.found;
+        return this.triggerFinderLineComment.found;
```

Comment:
Since we only have the one variable now, could this be renamed to `triggerFinderHelpComment` so that it's more apparent this applies to both block and line?